docs/changelog/0.4.0.md

```diff
@@ -0,0 +1,532 @@
+
+# 0.4.0 (Jan 14, 2021)
+
+We are happy to announce the next major release of Scala Native!
```

Comment:
```suggestion
We are happy to announce release of Scala Native 0.4.0!
```
"the next release" doesn't make much sense, because as soon as we announcing, it's not "next" anymore, it's "current". ;)